### PR TITLE
[gui] Select tag for each run filter item

### PR DESCRIPTION
**Pull request by csordasmarton**
Screenshot:
image

### CODE REVIEW

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/codechecker/pullRequest?prid=6274362&bid=20421848)

File: web/server/vue-cli/src/components/Report/ReportFilter/Filters/BaselineTagItems.vue [hunk 1]
```diff
@@ -1,83 +1,167 @@
...
+      const offset = 0;
+
+      reportFilter.runTag = await this.getRunTagFilter();
+
+      return new Promise(resolve => {
+        ccService.getClient().getRunHistoryTagCounts([ this.runId ],
+          reportFilter, null, limit, offset, handleThriftError(res => {
+            resolve(res.map(tag => {
+              const id = tag.id.toNumber();
+              const name = tag.name || "N/A";
```
Comment (gyorb) [inline web/server/vue-cli/src/components/Report/ReportFilter/Filters/BaselineTagItems.vue:141]: What do you think about renaming the `N/A` to something like `Storage without a named tag`
If I have multiple store events without a tag I just see a list of `N/A`.

File: web/server/vue-cli/src/components/Report/ReportFilter/Filters/BaselineRunFilter.vue [hunk 2]
```diff
@@ -20,79 +21,97 @@
...
+            :format="formatRunTitle"
             @apply="apply"
-            @cancel="cancel"
+            @cancel="cancelItemSelection"
             @select="select"
             @update:items="items.splice(0, items.length, ...$event)"
           >
+            <template v-slot:append-toolbar>
+              <bulb-message>
+                Use <v-icon>mdi-cog</v-icon> button beside each run after
```
Comment (gyorb) [inline web/server/vue-cli/src/components/Report/ReportFilter/Filters/BaselineRunFilter.vue:53]:
```suggestion
                Use the <v-icon>mdi-cog</v-icon> button beside each run after
```

**Review by gyorb — CHANGES_REQUESTED** [review]
If I have a store event on the 6th of October it is listed if I select some dates after that (7th or 8th ...) was there a discussion to see the history elements around that date (+- 5days) and not just after the selected date?

Another question if I only hover over a run (not select it) and I select a run history event (or tag) than click apply the run will not be selected and the reports will not be filtered based on that,

**Comment by gyorb** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CodeChecker/codechecker/pullRequest?prid=6274362&bid=20421848)

**Comment by csordasmarton** [comment]
@gyorb Yes, there was a dicussion but are we sure that we want to do that. Most of the time the user would like to find a tag by a date. For example last week Monday. I select the date end it will gives me the tags. The first item will be the latest tag on this date. So most of the time I will select the first item.
If we show the previous 5 tags and the next 5 tags for me it would be hard to find the tag what I am looking for.

**Review by gyorb — APPROVED** [review]

**Pull request merged**